jerry-debugger/jerry-client-ws.html

```diff
@@ -920,6 +927,33 @@
 
         return;
       }
+      case JERRY_DEBUGGER_OUTPUT_RESULT:
+      case JERRY_DEBUGGER_OUTPUT_RESULT_END:
+      {
+        var outputResult = concatUint8Arrays(outputResult, message);
+        var subType = outputResult[outputResult.length - 1];
+        outputResult = outputResult.slice(0, -1);
+        if (subType == JERRY_DEBUGGER_OUTPUT_OK)
+        {
+          appendLog("p_OUT: " + cesu8ToString(outputResult));
+          outputResult = null;
+          return;
+        }
+        else if (subType == JERRY_DEBUGGER_EVAL_WARNING)
+        {
+          appendLog("p_WARNING: " + cesu8ToString(outputResult));
+          outputResult = null;
+          return;
+        }
+        else if (subType == JERRY_DEBUGGER_OUTPUT_ERROR)
+        {
+          appendLog("p_ERROR: " + cesu8ToString(outputResult));
+          outputResult = null;
+          return;
```

Comment:
Also I would create a string variable, set its value in the `if` (or use a switch) and call `appendLog` `outputResult = null``at the end.